faunadb/client.py

```diff
@@ -177,8 +215,8 @@ def _execute(self, action, path, data=None, query=None):
     FaunaError.raise_for_status_code(request_result)
     return _get_or_raise(request_result, response_content, "resource")
 
-  def _perform_request(self, action, path, data, query):
+  def _perform_request(self, action, path, data, query, headers):
     """Performs an HTTP action."""
     url = self.base_url + "/" + path
-    req = Request(action, url, params=query, data=to_json(data), auth=self.auth)
+    req = Request(action, url, params=query, data=to_json(data), auth=self.auth, headers=headers)
```

Comment:
Is this adding new headers or overriding the session headers?

Reply:
session headers is stored in `self.session = Session()`

Reply:
Adds new one - per http://docs.python-requests.org/en/master/user/quickstart/ and confirmed in introspecting the traffic from running `tests.test_client.ClientTest.test_last_txn_time` - https://gist.github.com/dijkstracula/edcf89cd6e820b86718bf002fa50cae1 (note the absence of the header on the first request and the presence in all subsequent ones)